examples/thttpd/Makefile

```diff
@@ -31,6 +31,8 @@ ifeq ($(CONFIG_THTTPD_BINFS),y)
 else
   CONTENT_MAKE += -f Makefile.nxflat
   CSRCS += symtab.c
+  symtab.c_CFLAGS = -fno-builtin
```

Comment:
@xiaoxiang781216 maybe we also can use the similar for memcpy and other functions instead of adding an attribute that was added to compiler.h?